Mobile/ShopLens/Droid/SpeechActivity.cs

```diff
@@ -101,6 +101,7 @@ protected override void OnActivityResult(int requestCode, Result resultCode, Int
                     }
                     // change the text back on the button
                     recButton.Text = "Start Recording";
+                    isRecording = false;
```

Comment:
Now you don't have to click two times to start the voice recognizer again. At least on my device you don't.